### ISSUE TITLE
Minor/Major Translation issues/Yoigoroshi(Minor spoilers)

### ISSUE THREAD

**Issue by Mikulovers** [07th-mod/higurashi-console-arcs#42]
These are some issues with the translated text I had that I think could need some revision(If I am wrong then please do tell)
- [https://imgur.com/a/j6GRMFH](url) (Kinda unsure but I think she meant the thing they talked about and not like its written. Right now it looks kinda unnatural.

- [https://imgur.com/a/H44rQTS](url) Typo (The->They)

- [https://imgur.com/a/SfRiMHZ](url) "Maybe I should go prepared" didnt match the original meaning 

- [https://imgur.com/a/f9f587Z](url) "Dont go Alone" Around this part it felt kinda "wrong"

**Comment by Mikulovers** [comment]
Sorry for the bad links first time using github like this. just copy paste them into browser.

**Comment by ItaloKnox** [comment]
Just as a heads up if you plan to post more issues later (please do!), you can embed images by using this format:

``![caption](link)``
You can skip the caption if you want, just use ``![](link)`` for that.

To create regular links, use the same format but remove the ``!`` at the start. Don't forget that you need the caption for it to work, since that becomes the link itself. If I wanted to link the first image, I could do it like [this](https://imgur.com/a/j6GRMFH). It would look like ``[this](https://imgur.com/a/j6GRMFH)`` when writing the comment.

I hope it was easy to understand, thanks for reporting these issues. @DoctorDiablo will take care of them. Please report more if you see them! 👍

**Comment by DoctorDiablo** [comment]
Thanks for the help!  I just pushed a fix for those issues.  If you find any more, feel free to post them here.

**Comment by Mikulovers** [comment]
Here are a couple of more that could need revision:
1 ("I wont let you say you dont know" -> "If you really dont know why arent you saying something"?)
Also the second she says "Im the one asking" but it got translated as "Im listening"?
![2wrong](https://user-images.githubusercontent.com/39219285/52972831-57b72780-33bc-11e9-91f8-ea32ce8fa83d.png)
2 The part about the regret.
![regret](https://user-images.githubusercontent.com/39219285/52972881-86350280-33bc-11e9-8b0c-bbb7e8a3dd7d.png)
3 (This one you might have had the intention to omit some parts but I mention it anyway)
![sometinglikethat](https://user-images.githubusercontent.com/39219285/52972914-a238a400-33bc-11e9-8fb1-5c501795e10e.png)
4 The Spy comment is completely off
![takumispy](https://user-images.githubusercontent.com/39219285/52972949-c5fbea00-33bc-11e9-967f-b60a92339539.png)
5 At first I thought this was wrong, but now I am unsure. But I might as well post so you can check if it fits in the context while you are at it.
![overthinking](https://user-images.githubusercontent.com/39219285/52973191-8d104500-33bd-11e9-8d69-5ed70ca6855e.png)

**Comment by Mikulovers** [comment]
This one struck me a bit but I might be wrong. Is she not referring to Miyuki also being in the way in the japanese sentence?
![in the way](https://user-images.githubusercontent.com/39219285/53093635-3ff4b600-3518-11e9-862d-2ae8d44c58a9.png)

**Comment by Mikulovers** [comment]
Most likely my final post for this particular chapter as I finished reading it now.
1. While still being understandable, the second one just feels a bit unnatural, maybe "You wont get off easy for...?"
![the boss](https://user-images.githubusercontent.com/39219285/53191238-97278300-360b-11e9-8114-7c61bd2e1142.png)
2. The Japanese in this context I think refers to the danger she thinks she put him in so the English translation feels a bit off. Maybe something like "I'm sorry I had to put you through this." would work?
![danger](https://user-images.githubusercontent.com/39219285/53191431-16b55200-360c-11e9-9991-b08cb064f5af.png)
